web/src/beta/features/Modals/WorkspaceCreationModal/index.tsx

```diff
@@ -100,7 +100,7 @@ const StyledInput = styled.input`
   color: ${props => props.theme.content.main};
   background: ${props => props.theme.bg[0]};
   border-radius: 4px;
-  border: 1px solid #3f3d45;
+  border: 1px solid ${props => props.theme.bg[4]};
```

Comment:
please keep coding style consistent. Please destructure theme here 🙏🏻

Reply:
sorry i didn't get it , could you explain more

Reply:
done